src/aragonjs-wrapper.js

```diff
@@ -323,6 +325,22 @@ const subscribe = (
   return subscriptions
 }
 
+const hydrateCacheFromFeather = async (daoAddress, cache) => {
```

Comment:
Amazed at how simple this is :)

As discussed offline, I'd rather have this built into aragon.js directly so it can be used as a standalone library if initialized with a cache server!

Reply:
Yeah, I'll look in to adding it to Aragon.js and open up new PRs with those changes in both repositories when I get some time 😊